{"name": "Login with wrong password", "status": "passed", "description": "Signing in with the following credentials email : kamelmohsenkamel@gmail.com & password: wrong password", "attachments": [{"name": "test_login_failed_login", "source": "c39f1824-dfaf-446c-998f-74002f69ab01-attachment.png", "type": "image/png"}], "start": 1583929529763, "stop": 1583929532431, "uuid": "4ddc7557-00c4-4eb2-ad24-604861bc9aec", "historyId": "9002a5477b70c2d2732bd4879bd4c9b3", "testCaseId": "5849ff08927baae0bfe7bd1d291d6dac", "fullName": "Tests.login_tests.Test#test_case_2", "labels": [{"name": "severity", "value": "blocker"}, {"name": "subSuite", "value": "Login with wrong password"}, {"name": "feature", "value": "Log in Tests"}, {"name": "suite", "value": "Login Tests"}, {"name": "parentSuite", "value": "End to End testing"}, {"name": "story", "value": "Failing login"}, {"name": "tag", "value": "Do"}, {"name": "tag", "value": "Login"}, {"name": "host", "value": "DESKTOP-1RVKD4K"}, {"name": "thread", "value": "21024-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "Tests.login_tests"}]}